src/Client.php

```diff
@@ -79,11 +82,13 @@ public function captureMessage(string $message, ?Severity $level = null, ?Scope
             'level' => $level,
         ];
 
-        if ($event = $this->prepareEvent($payload, $scope, $this->options->shouldAttachStacktrace())) {
-            return $this->transport->send($event);
+        $event = $this->prepareEvent($payload, $scope, $this->options->shouldAttachStacktrace());
+
+        if (null === $event) {
+            return null;
         }
 
-        return null;
+        return $this->transport->send($event);
```

Comment:
I know this wasn't there before, but I would really appreciate it if we have the `transport->send` only once. Meaning in all other functions like `captureMessage` we call `$this->captureEvent` in the end.
And only `captureEvent` invokes the transport.

Reply:
We can't because we need to call `prepareEvent` with the 3rd argument specified while `captureEvent` does it without it. This is the reason it was done like this even before